parse/ShipPartsParser.cpp

```diff
@@ -1,15 +1,10 @@
+#define FUSION_MAX_VECTOR_SIZE 20
```

Comment:
It may be necessary to build Boost with this define set differently, not just define it in a single file. Which would mean you can't change it, as most people won't have built boost with a different value set.

Reply:
So in short i think the define is not for building boost but for configuring it. (But i my be wrong)

Reply:
I didn't look into that define specifically, but at least some other similar macros worked that way.

Reply:
As far as i understand after reading up on building with boost, the used stuff is mostly headers. Especially it seems fusion and spirit are headers-only and don't require a build. Can you remember an example for the define you mentioned? Maybe serialization?

Reply:
Oh, CI went fine with the manual fusion_max_vector_size :)